.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @ancostas @dbradf @lydiastepanek @jimoleary @varsha804 @zamj
+* @ancostas @dbradf @lydiastepanek @jimoleary @varsha804 @zamj
```

Comment:
I don't see any changes to this file?

Reply:
Reverted. (I wasn't sure why @varsha804 wasn't getting added as a Reviewer, so I tried removing the new line at the end of this file. I think actually the problem was that I needed to add her as a [collaborator](https://github.com/mongodb/selected-tests/settings/collaboration).)